adapter/pkg/logging/logging_constant.go

```diff
@@ -339,6 +348,16 @@ var Mapper = map[int]ErrorDetails{
 		Message:   "Error while getting custom rate limit policies: %s",
 		Severity:  MAJOR,
 	},
+	error2651: {
+		ErrorCode: error2651,
+		Message:   "Error while getting interceptor service %s, %s",
+		Severity:  BLOCKER,
```

Comment:
is this a blocking error?

Reply:
this error is when a api server connectivity or its internal error scenario, so kept as a blocker

Reply:
changed to `CRITICAL` severity level